hs_tools_resource/forms.py

```diff
@@ -63,6 +63,32 @@ class Meta:
 class VersionValidationForm(forms.Form):
     value = forms.CharField(max_length=128)
 
+
+class ToolIconFormHelper(BaseFormHelper):
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None, *args, **kwargs):
+        # the order in which the model fields are listed for the FieldSet is the order these fields will be displayed
+        field_width = 'form-control input-sm'
+        layout = Layout(
+                Field('icon', css_class=field_width),
+        )
+        kwargs['element_name_label'] = 'Tool Icon URL'
```

Comment:
I am not sure how this works, hence naive question: do you allow users to edit tool icon url? This FormHelper class seems to indicate it is editable. If this is indeed the case, could you elaborate a bit on why tool icon url is editable or generically what the tool icon url editing use case looks like?